saas/urls/api/users.py

```diff
@@ -28,13 +28,16 @@
 
 from django.conf.urls import url
 
-from ...api.roles import AccessibleByListAPIView, RoleDetailAPIView
+from ...api.roles import (AccessibleByListAPIView, RoleDetailAPIView,
+    RoleAcceptAPIView)
 from ...api.users import AgreementSignAPIView
 from ... import settings
 
 urlpatterns = [
     url(r'^legal/(?P<agreement>%s)/sign/$' % settings.ACCT_REGEX,
         AgreementSignAPIView.as_view(), name='saas_api_sign_agreement'),
+    url(r'^users/(?P<user>%s)/accessibles/accept/' % settings.MAYBE_EMAIL_REGEX,
```

Comment:
The `verification_key` must be passed on the URL and not as a data parameter, otherwise we have an API which is not consistent with the other endpoints dealing with double opt-in. See
```    url(r'^profile/(?P<organization>%s)/subscribers/accept/'\
        '(?P<request_key>%s)/' % (ACCT_REGEX, VERIFICATION_KEY_RE),
        SubscriptionRequestAcceptAPIView.as_view(),
        name='saas_api_subscription_grant_accept'),
```